cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,498 @@
+                    raise ExecutionFailureException(
+                        ("Provided 'data-pool-name': '{}', " +
+                         "doesn't match from the data-pools' list: {}").format(
+                            self._arg_parser.cephfs_data_pool_name,
+                            [str(x) for x in matching_json_out['data_pools']]))
+            else:
+                self._arg_parser.cephfs_data_pool_name = str(
+                    matching_json_out['data_pools'][0])
+            if len(matching_json_out['data_pools']) > 1:
+                print("WARNING: Multiple data pools detected.\n" +
+                      "{}\n".format([str(x) for x in matching_json_out['data_pools']]) +
+                      "Using the data-pool: {}\n".format(self._arg_parser.cephfs_data_pool_name))
+
+    def create_cephCSIKeyring_RBDNode(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": "client.csi-rbd-node",
+                    "caps": ["mon", "profile rbd",
+                             "osd", "profile rbd"],
+                    "format": "json"}
+        ret_val, json_out, err_msg = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0 or len(json_out) == 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create client.csi-rbd-node' command failed\n" +
+                "Error: {}".format(err_msg if ret_val != 0 else self.EMPTY_OUTPUT_LIST))
+        return str(json_out[0]['key'])
+
+    def create_checkerKey(self):
+        cmd_json = {"prefix": "auth get-or-create",
+                    "entity": self.run_as_user,
+                    "caps": ["mon", "allow r, allow command quorum_status",
+                             "osd", ("allow rwx pool={0}.rgw.meta, " +
+                                     "allow r pool=.rgw.root, " +
+                                     "allow rw pool={0}.rgw.control, " +
+                                     "allow x pool={0}.rgw.buckets.index").format(self._arg_parser.rgw_pool_prefix)],
+                    "format": "json"}
+        # allow rwx pool='"$RGW_POOL_PREFIX"'.rgw.meta, allow r pool=.rgw.root, allow rw pool='"$RGW_POOL_PREFIX"'.rgw.control, allow x pool='"$RGW_POOL_PREFIX"'.rgw.buckets.index'
+        ret_val, json_out, err_msg = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0 or len(json_out) == 0:
+            raise ExecutionFailureException(
+                "'auth get-or-create {}' command failed\n".format(self.run_as_user) +
+                "Error: {}".format(err_msg if ret_val != 0 else self.EMPTY_OUTPUT_LIST))
+        return str(json_out[0]['key'])
+
+    def _gen_output_map(self):
+        if self.out_map:
+            return
+        self.get_cephfs_data_pool_details()
+        self.out_map['NAMESPACE'] = self._arg_parser.namespace
+        self.out_map['CLUSTER_NAME'] = self._arg_parser.cluster_name
```

Comment:
`CLUSTER_NAME` shouldn't be printed on the bash output. Just `NAMESPACE`.

Reply:
Added in the `_excluded_keys`, which won't be printed in the output